src/Container/Container.php

```diff
@@ -252,6 +252,9 @@ public function resolveParameters(array $parameters, array $predefinedValues = [
          */
         foreach ($parameters as $parameter) {
             if (\array_key_exists($parameter->getName(), $predefinedValues)) {
+                if ($parameter->getType()->isBuiltin()) {
```

Comment:
```suggestion
                if ($parameter->getType() && $parameter->getType()->isBuiltin()) {
```
Since `$parameter->getType()` can return `null` if no type is specified. What do you think?